app/components/documents/document-preview/document-preview-sidebar.hbs

```diff
@@ -73,25 +73,31 @@
       <Utils::LoadableContent @isLoading={{this.loadPieceData.isRunning}}>
         {{! DETAILS }}
         {{#if (eq this.tab "details")}}
-          <Documents::DocumentPreview::DetailsTab
-            @piece={{@piece}}
-            @documentContainer={{this.documentContainer}}
-            @didDeletePiece={{@didDeletePiece}}
-            @onChangeFile={{@onChangeFile}}
-          />
-          {{! SIGNATURES }}
+          <div class="auk-u-m-4">
+            <Documents::DocumentPreview::DetailsTab
+              @piece={{@piece}}
+              @documentContainer={{this.documentContainer}}
+              @didDeletePiece={{@didDeletePiece}}
+              @onChangeFile={{@onChangeFile}}
+            />
+            {{! SIGNATURES }}
+          </div>
         {{else if (eq this.tab (t "signatures"))}}
-          <Documents::DocumentPreview::SignaturesTab
-            @piece={{@piece}}
-            @documentContainer={{this.documentContainer}}
-          />
-          {{! VERSIONS }}
+          <div class="auk-u-m-4">
+            <Documents::DocumentPreview::SignaturesTab
+              @piece={{@piece}}
+              @documentContainer={{this.documentContainer}}
+            />
+            {{! VERSIONS }}
+          </div>
         {{else if (eq this.tab (t "versions"))}}
-          <Documents::DocumentPreview::VersionsTab
-            @documentContainer={{this.documentContainer}}
-            @currentVersion={{@piece}}
-            @onSelectVersion={{@onSelectVersion}}
-          />
+          <div class="auk-u-m-4">
```

Comment:
This was `auk-u-m-2` inside `Documents::DocumentPreview::VersionsTab`. Looks fine like this too, just making sure it's an intentional change.

Reply:
This was indeed an intentional change (to sync up the spacing with the other tabs) ✅